### PR TITLE
Add item published and updated date

### PR DESCRIPTION
**Pull request by crazy-max**
Linked to Issue #267

### CODE REVIEW

File: lib/PicoFeed/Parser/Rss10.php [hunk 1]
```diff
@@ -158,17 +158,29 @@ public function findFeedDate(SimpleXMLElement $xml, Feed $feed)
...
+    /**
+     * Find the item updated date.
+     *
+     * @param SimpleXMLElement      $entry Feed item
+     * @param Item                  $item  Item object
+     * @param \PicoFeed\Parser\Feed $feed  Feed object
+     */
+    public function findItemUpdatedDate(SimpleXMLElement $entry, Item $item, Feed $feed)
+    {
+        $item->setUpdatedDate(null); // No updated date in RSS 1.0 specifications
```
Comment (BernhardPosselt) [inline lib/PicoFeed/Parser/Rss10.php]: Should be the same as the published date.

File: lib/PicoFeed/Parser/Rss20.php [hunk 2]
```diff
@@ -153,17 +153,29 @@ public function findFeedDate(SimpleXMLElement $xml, Feed $feed)
...
+    /**
+     * Find the item updated date.
+     *
+     * @param SimpleXMLElement      $entry Feed item
+     * @param Item                  $item  Item object
+     * @param \PicoFeed\Parser\Feed $feed  Feed object
+     */
+    public function findItemUpdatedDate(SimpleXMLElement $entry, Item $item, Feed $feed)
+    {
+        $item->setUpdatedDate(null); // No updated date in RSS 2.0 specifications
```
Comment (BernhardPosselt) [inline lib/PicoFeed/Parser/Rss20.php]: Should be the same as the published date.

File: lib/PicoFeed/Parser/Rss10.php (hunk 1 above)
Comment (crazy-max) [inline lib/PicoFeed/Parser/Rss10.php]: UpdatedDate should be the same as PublishedDate for RSS10 and RSS20 ?

Comment (BernhardPosselt) [inline lib/PicoFeed/Parser/Rss10.php]: Yes, basically: 
```php
$item->setUpdatedDate($item->getPublishedDate())
```

Needs to be run after getting the published date ofc :)

Comment (crazy-max) [inline lib/PicoFeed/Parser/Rss10.php]: Yep published date is already retrieved before updated date : https://github.com/crazy-max/picoFeed/blob/master/lib/PicoFeed/Parser/Parser.php#L224

Btw you are working on ownCloud News to include the enhancement ?

Comment (BernhardPosselt) [inline lib/PicoFeed/Parser/Rss10.php]: I will once it's merged unless you want to work on it ;)

Comment (crazy-max) [inline lib/PicoFeed/Parser/Rss10.php]: Ahahah yes why not but i prefer to wait the merge like you o:)

**Comment by BernhardPosselt** [comment]
Very small todo: we should also adjust the docs here :)  @crazy-max can you take care of that :)?

https://github.com/fguillot/picoFeed/blob/master/docs/feed-parsing.markdown#feed-and-item-properties

**Comment by crazy-max** [comment]
Ok no problem @BernhardPosselt ;)

**Pull request merged**